examples/5_validations.md

```diff
@@ -0,0 +1,137 @@
+
+## Integer
+
+Keyword `Integer` checks the expected field as JSON [integer](https://en.wikipedia.org/wiki/Integer)
+
+Example usages:
+
+| Keyword | Field | Enum | Validations | Remarks |
+| ------- | ----- | ---- | ----------- | ------- |
+| `GET`  | /users/1 | | | # https://jsonplaceholder.typicode.com/users/1 |
+| `Integer` | response body id | | | # value is optional |
+| `Integer` | response body id | 1 |
+| `Integer` | response body id | ${1} | | # same as above |
+| `Integer` | $.id | 1 | | # JSONPath alternative |
+| `GET`  | /users?_limit=10 | | | # https://jsonplaceholder.typicode.com/users |
+| `Integer` | response body 0 id | 1 | | |
+| `Integer` | $[0].id | 1 | | # same as above |
+| `Integer` | $[*].id | | minimum=1 | maximum=10 |
+
+## Number
+
+Keyword `Number` checks the expected field as JSON number.
+
+Example usages:
+
+| Keyword | Field | Enum | Validations | Remarks |
+| ------- | ----- | ---- | ----------- | ------- |
+| `PUT`  | /users/1 | { "allocation": 95.0 } | | # https://jsonplaceholder.typicode.com/users/1 |
+| `Number` | response body allocation | | | # value is optional |
+| `Number` | response body allocation | 95.0 |
+| `Number` | response body allocation | ${95.0} |  |# same as above |
+| `Number` | $.allocation | 95.0 | | # JSONPath alternative |
+| `GET`  | /users?_limit=10 | | | # https://jsonplaceholder.typicode.com/users |
+| `Number` | $[0].id | 1 | | # integers are also numbers |
+| `Number` | $[*].id | | minimum=1 | maximum=10 |
+
+## String
+
+Keyword `String` checks the expected field as JSON string.
+
+Example usages:
+
+| Keyword | Field | Enum | Validations | Remarks |
+| ------- | ----- | ---- | ----------- | ------- |
+| `GET`  | /users/1 | | | # https://jsonplaceholder.typicode.com/users/1 |
+| `String` | response body email | | | # value is optional |
+| `String` | response body email | Sincere@april.biz |
+| `String` | $.email | Sincere@april.biz | | # JSONPath alternative |
+| `String` | $.email | | format=email |
+| `GET`  | /users?_limit=10 | | | # https://jsonplaceholder.typicode.com/users |
+| `String` | response body 0 email | Sincere@april.biz |
+| `String` | $[0].email | Sincere@april.biz | | # same as above |
+| `String` | $[*].email | | format=email |
+
+## Object
+
+Keyword `Object` checks the expected field as JSON object.
+
+Example usages:
+
+| Keyword | Field | Enum | Validations | Remarks |
+| ------- | ----- | ---- | ----------- | ------- |
+| `GET`  | /users/1 | | | # https://jsonplaceholder.typicode.com/users/1 |
+| `Object` | response body | |
+| `Object` | response body | required=["id", "name"] | | # can have other properties |
+| `GET`  | /users/1 | | | # https://jsonplaceholder.typicode.com/users/1 |
+| `Object` | $.address.geo | required=["lat", "lng"] |
+| `Object` | $..geo | additionalProperties=false | | # cannot have other properties |
+
+## Array
+
+Keyword `Array` checks the expected field as JSON array.
```

Comment:
Maybe add a remark that they keyword doesn't validate the array _contents_, just that given JSONPath is an array.

Reply:
Excellent point. I'll add that.